.env.dev

```diff
@@ -0,0 +1,4 @@
+
+###> symfony/framework-bundle ###
+APP_SECRET=3f4c32c6ad4e044999f19d9b97a0df9d
```

Comment:
Yes the bundler did add this secret for the `.env.dev` by default. And it doesn't seem to be that we should ignore this `.env.dev` file for Symfony.

Reply:
And if you do not believe me, you can create a fresh symfony project. And look at the file structure and `.env` files, including the `.gitignore`..

```sh
symfony new my_project_directory --version="7.2.x" --webapp
```

Reply:
In fact.. we should actually also check-in the `.env` file and only use `.env.local` to make additional changes that deviate from the defaults, like secrets, etc.